README.md

```diff
@@ -131,6 +131,11 @@ Parse Dashboard is continuously tested with the most recent releases of Node.js
 | Parameter                              | Type                | Optional | Default | Example              | Description                                                                                                                                 |
 |----------------------------------------|---------------------|----------|---------|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
 | `apps`                                 | Array&lt;Object&gt; | no       | -       | `[{ ... }, { ... }]` | The apps that are configured for the dashboard.                                                                                             |
+| `apps.appId`                           | String                        | yes          | -           | `"myAppId"`            | The Application ID for your Parse Server instance.                                                                                          |
+| `apps.masterKey`                       | String / () => string         | yes          | -           | `"myMasterKey"` or `() => "myMasterKey"` | The master key for full access to your Parse Server. It can be provided directly as a string or as a function returning a string.        |
+| `apps.masterKeyTtl`                    | Number                        | no           | -           | `3600`                  | Time-to-live (TTL) for the master key in seconds. This defines how long the master key is cached before the `masterKey` function is re-triggered.                |
```

Comment:
```suggestion
| `apps.masterKey`                       | String \| Function         | yes          | -           | `"exampleMasterKey"`, `() => "exampleMasterKey"` | The master key for full access to Parse Server. It can be provided directly as a String or as a Function returning a String.        |
| `apps.masterKeyTtl`                    | Number                        | no           | -           | `3600`                  | Time-to-live (TTL) for the master key in seconds. This defines how long the master key is cached before the `masterKey` function is re-triggered.                |
```